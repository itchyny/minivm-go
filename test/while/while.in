a = 10
b = 10
while a > 0
  print a * b
  a = a - 1
  b = b + 1
end

c = 5
while c > 0
  d = 5
  while d > 0
    print c * d
    d = d - 1
  end
  c = c - 1
end
print c
