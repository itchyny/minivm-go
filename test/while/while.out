100
99
96
91
84
75
64
51
36
19
25
20
15
10
5
20
16
12
8
4
15
12
9
6
3
10
8
6
4
2
5
4
3
2
1
0
